test/k8sT/Policies.go

```diff
@@ -364,6 +357,17 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 				helpers.DefaultNamespace, appPods[helpers.App3],
 				helpers.CurlFail(fmt.Sprintf("http://%s/public", clusterIP)))
 			res.ExpectFail("Ingress connectivity should be denied by policy")
+
+			Expect(kubectl.WaitCEPReady()).To(BeNil(), "Cep is not ready in a timeout")
+			for _, app := range []string{helpers.App1, helpers.App2, helpers.App3} {
```

Comment:
Add a `By` indicating that we are checking that ingress policy enforcement is enabled on all pods